db/models.py

```diff
@@ -1 +1,10 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
```

Comment:
Required correct display of objects. Your code result / Required in README
![image](https://github.com/mate-academy/py-genres-and-actors/assets/106102798/aca06712-b71d-4177-9cf4-84382d02ca00)
![image](https://github.com/mate-academy/py-genres-and-actors/assets/106102798/7decfaad-b9da-4cfb-9eb7-ea8e1b5cde72)